2021-12-16 19:36:29.550002 | API is disabled on startup
2021-12-16 19:36:29.551003 | Settings loaded!
2021-12-16 19:36:30.519003 | Started VRCA loading process...
2021-12-16 19:36:30.520004 | Propting user to select VRCA...
2021-12-16 19:36:33.111493 | No VRCA selected, load cancelled!
2021-12-16 19:36:33.853027 | Started repair process...
2021-12-16 19:36:33.854028 | Propting user to select VRCA...
2021-12-16 19:36:36.222197 | VRCA selected: C:/Users/adamc/Downloads/Avatar-Pandas-Demon-Compendium-.file_e3f724e6-61fd-4aec-8ec5-a0e47e5e27c7.2_1.vrca
2021-12-16 19:36:36.223198 | Starting repair on new thread...
2021-12-16 19:36:36.224199 | Error occured during repair process!
Traceback (most recent call last):
  File "C:/Users/adamc/Documents/GitHub/A.R.E.S/ARES(rewritten)/GUI.py", line 229, in RepairVRCAWrapper
    threading.Thread(target=RepairVRCA, args={QtWidgets.QProgressBar.self.ProgBar,QtWidgets.QPushButton.self.RepairVRCA}).start()
AttributeError: type object 'QProgressBar' has no attribute 'self'

